TodoListClient/.eslintrc.js

```diff
@@ -24,15 +27,16 @@ module.exports = {
     // node files
     {
       files: [
-        '.eslintrc.js',
-        '.prettierrc.js',
-        '.template-lintrc.js',
-        'ember-cli-build.js',
-        'testem.js',
-        'blueprints/*/index.js',
-        'config/**/*.js',
-        'lib/*/index.js',
-        'server/**/*.js',
+        './.eslintrc.js',
+        './.prettierrc.js',
+        './.stylelintrc.js',
+        './.template-lintrc.js',
+        './ember-cli-build.js',
+        './testem.js',
+        './blueprints/*/index.js',
+        './config/**/*.js',
+        './lib/*/index.js',
+        './server/**/*.js',
```

Comment:
If I run `ember-cli-update --to 3.28` on the master branch, I'm not getting these changes. Note I'm okay with changing these; just wondering where they came from and if they are intentional.

I'm also getting slightly different versions in package.json, compared to the first commit in this PR. Am I doing something wrong?

Reply:
@briarsweetbriar I removed the vscode configs earlier, because they were outdated and I didn't have vscode installed. It's nice to have working settings again, so let's keep them in.

Reply:
These changes go all the way to 4.12 (despite the branch name, sorry if that caused confusion). They're generated by ember-cli-update.